config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  root 'static_pages#home'
+  get  '/signup'  => 'users#new'
+  post '/signup'  => 'users#create'
+  resources :users
```

Comment:
Why do we still need this line when we already have lines 3-4 to route us to the signup page and route to the create user action?